lib/usd/pxrUsdPreviewSurface/plugInfo.json

```diff
@@ -11,10 +11,9 @@
                     }
                 }
             },
-            "Name": "pxrUsdPreviewSurface",
-            "ResourcePath": "@PLUG_INFO_RESOURCE_PATH@",
-            "Root": "@PLUG_INFO_ROOT@",
-            "Type": "resource"
+            "LibraryPath": "../../../@CMAKE_SHARED_LIBRARY_PREFIX@@TARGET_NAME@@CMAKE_SHARED_LIBRARY_SUFFIX@",
+            "Name": "@TARGET_NAME@",
+            "Type": "library"
         }
```

Comment:
Based on plugInfo.json in lib/usd/translators.

Reply:
I think the change in structure actually means that you could leave the `mayaPlugin` field above empty. The node and the translator are now getting bundled into the `mayaUsd` library itself, right?

https://github.com/Autodesk/maya-usd/blob/1a5d92637fedcfbf8ecf53b90f17e8081b65e17d/lib/mayaUsd/fileio/registryHelper.cpp#L170

The `mayaUsd` library itself is what provides the translator, so it's not actually necessary to load any plugins to ensure that the translator has been registered, since the library will already have been loaded.

As it is now, I think if you only had the adsk `mayaUsdPlugin` loaded and you did an export, it would needlessly load the pxr `pxrUsdPreviewSurface` plugin thinking it needs to do so in order to make sure the translator is available.

Reply:
@mattyjams We want to keep the original plugin structure for preview surface, i.e. the node and the translator is not part of `mayaUsd` library. This is why we put it in `lib/usd` and not  `lib/mayaUsd`.

What Jerry talks about is the current limitation where you can have only one translator registered per node type. We will need to support more than one.

As for line 7 - pxrUsdPreviewSurface is a plugin that shouldn't be dependent on ADSK, AL, nor PXR plugins. Why would we need to put `mayaUsdPlugin` in there?

Reply:
Indeed the plugInfo.json file is not necessary. The writer will get registered as soon as the basePxrUsdPreviewSurface DLL gets loaded in memory, which will happen as soon as one of the two plugins using UsdPreviewSurface gets loaded.

Reply:
The plugInfo.json is probably not strictly necessary for export, no, since as you mentioned, the library that provides the translator will already have been loaded to make the node itself available.

It would definitely be necessary for import though, albeit that's not implemented yet. You'll need to `loadPlugin` *something* in order to have that library be loaded in the case where you're importing into an empty scene and the importer encounters a shader of that type. And if we're going to (eventually) include an entry in the plugInfo.json for import, it would be nice to have an entry for export as well, just for completeness.

`pxrUsdPreviewSurface` will live in `lib/usd` with this change, but what could cause that library to be loaded other than doing a `loadPlugin` of either `mayaUsdPlugin` or `pxrUsd`?